src/components/Header.js

```diff
@@ -1,46 +1,49 @@
 import React, { Component } from 'react';
 import { Localized } from 'fluent-react/compat';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';
 
 import './Header.css';
 
 class Header extends Component {
   render() {
     return (
-      <header>
-        <div className="container">
-          <a className="site-title" href="/">
-            <img src="/activate.png" alt="" className="brand" />
-          </a>
-
-          <button className="navbar-toggle collapsed"
-                  type="button"
-                  data-toggle="collapse"
-                  data-target="#CollapsingNavbar"
-                  aria-controls="CollapsingNavbar"
-                  aria-expanded="false"
-                  aria-label="Toggle navigation">
-            &#9776;
-          </button>
-
-          <div className="trigger collapse navbar-collapse" id="CollapsingNavbar">
+      <Navbar collapseOnSelect>
+        <Navbar.Header>
+          <Navbar.Brand>
+          <img src="/activate.png" alt="" className="brand" />
+          </Navbar.Brand>
+          <Navbar.Toggle />
+        </Navbar.Header>
+        <Navbar.Collapse>
+          <Nav>
             <Localized id="nav-activities">
-              <a className="page-link" href="/activities/">Activities</a>
+              <NavItem className="page-link" href="/activities/">
+              Activities
+              </NavItem>
             </Localized>
             <Localized id="nav-event-guide">
-              <a className="page-link" href="/eventguide/">Event Guide</a>
+              <NavItem className="page-link" href="/eventguide/">
+              Activities2
```

Comment:
minor: can you set this to "Event Guide"? It currently still works since it gets translated, but we need to keep in mind the case where translations can't be downloaded for some reason.

Reply:
Fixed! https://github.com/kelimuttu/activate-rewrite/commit/109aea310d9c4fb858557bd06725bf38b12031b1